src/main/java/salesforce/lightning/WorkTypeForm.java

```diff
@@ -0,0 +1,71 @@
+package salesforce.lightning;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import salesforce.BasePage;
+
+public class WorkTypeForm extends BasePage {
+
+    @FindBy(css = ".select[aria-required='true']")
+    protected WebElement estimatedDurationComboBox;
+
+    @FindBy(xpath = "//button[@data-aura-class='uiButton forceActionButton'][3]")
```

Comment:
Try to avoid using indexing, objects could change its place

Reply:
Is dynamic the text Box, not depend of index of posicion but of when it is created, I dont find other way to grab it